packages/server/src/routes/grants.js

```diff
@@ -235,14 +235,16 @@ router.delete('/:grantId/interested/:agencyId', requireUser, async (req, res) =>
     const { user } = req.session;
     const { grantId, agencyId } = req.params;
     const { agencyIds } = req.body;
-    console.log(agencyIds);
-    const allowed = await isUserAuthorized(user, agencyIds || agencyId);
+    // If agencyIds is not empty, use that. Otherwise, use the single agencyId value.
+    const selectedAgencyIds = agencyIds.length > 0 ? agencyIds : [agencyId];
```

Comment:
nit: I'd name this something else because `selectedAgencyId` has a specific meaning: the agency that an admin is acting as (see `requireAdminUser`)